docs/contributing/pair-programming.md

```diff
@@ -6,25 +6,22 @@ The best part of open source is the community, and every community is stronger w
 
 ## How community pair programming sessions work
 
-1.  Sign up for any available slot on [the open pairing calendar][cal]
-2.  You’ll be paired with one of the Gatsby Inkteam
-3.  Join the video meeting during your time slot
-4.  Build, learn, or discuss your Gatsby or other OSS project!
+1.  Select or create [an issue](https://github.com/gatsbyjs/gatsby/issues) you would like to work on and work on it yourself.
+2.  Create a [Pull Request](https://github.com/gatsbyjs/gatsby/compare) when you encounter any problem.
+3.  Sign up for any available slot on [the open pairing calendar][cal]
+4.  You’ll be paired with one of the Gatsby Inkteam
+5.  Join the video meeting during your time slot (we are using Zoom)
+6.  We'll help you with your PR or Issue!
 
 ### What we expect in pair programming sessions
 
-These sessions are intended for people who:
-
-- would like to get started with Gatsby, or
-- would like to get started with contributing to open source software, or
-- would like to work on an issue or pull request related to Gatsby, or
-- are using Gatsby for a personal, open source, charity or education project
+These sessions are intended for people who would like to contribute to Gatsby. We will pair program together to solve an issue or pull request related to Gatsby.
 
 If you're interested in support for a commercial Gatsby project, please [get in touch via the contact page](https://www.gatsbyjs.com/contact-us/).
 
 We also expect the following from pair programming participants:
 
-- These sessions tend to work best when you have a specific goal for the session.
+- These sessions tend to work best when you have a specific goal for the session. Preferably you have already selected an issue or have started work and created a work-in-progress pull request.
```

Comment:
I think we are making it a requirement (not optional) that someone chooses an issue or PR?

```suggestion
- These sessions work best when you have a specific goal for the session. You should choose an issue or pull request that you'd like to work on.
```